actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorEntry.java

```diff
@@ -166,11 +166,11 @@ public T getObject()
             {
                 return Task.done();
             }
-            return executionSerializer.offerJob(key, () -> doDeactivate(), 10000);
+            return executionSerializer.offerJob(key, this::doDeactivate, 10000);
         }
         catch (final Throwable ex)
         {
-            // this should never happen, but deactivate must't fail.
+            // this should never happen, but deactivate must not fail.
             ex.printStackTrace();
```

Comment:
yeah, we have uncovered some exceptions being hidden by orbit in previous pull requests, so whenever I see this I always feel like we should be sending them to the logger.  I like it.